src/styles/dotcms-theme/components/_datatable.scss

```diff
@@ -69,12 +78,12 @@
 }
 
 .p-datatable .p-sortable-column:not(.p-highlight):hover {
-    background: $hover-bg;
+    background: rgba(var(--color-sec_rgb), 0.05);
```

Comment:
`$brand-secondary_rgb` instead of `--color-sec_rgb` ? Why not move this to a SCSS variable? I'm seeing below more `$brand-secondary_rgb, 0.05`

Reply:
Not sure how to handle this one because is like the hover and focus state just for the table and I don't see us using it anywhere else... do you think it would be good to have the var inline in this same file?

Reply:
What if we do:

```scss
@use 'variables' as * with (
  $hover-bg: rgba($brand-secondary_rgb, 0.05),
);
```

Not sure if `as * with` works though, but it's worth a try.